filebeat/module/postgresql/_meta/kibana/default/dashboard/Filebeat-Postgresql-overview.json

```diff
@@ -6,13 +6,13 @@
         "kibanaSavedObjectMeta": {
           "searchSourceJSON": "{\"filter\":[]}"
         },
-        "savedSearchId": "ef9da9b0-776c-11e7-9e03-393c0d1aa325",
+        "savedSearchId": "PostgreSQL All Logs",
```

Comment:
Just for my understanding, were the changes in the vis and searches IDs also necessary? Or would have the index pattern change be enough, and this is only for consistency?